src/tree/StepTreeItem.ts

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as moment from 'moment';
+import { AzureTreeItem, TreeItemIconPath } from "vscode-azureextensionui";
+import { Conclusion, Status } from '../constants';
+import { convertConclusionToVerb, convertStatusToVerb } from '../utils/gitHubUtils';
+import { getTimeElapsedString } from '../utils/timeUtils';
+import { treeUtils } from "../utils/treeUtils";
+import { IAzureResourceTreeItem } from './IAzureResourceTreeItem';
+import { JobTreeItem } from './JobTreeItem';
+
+export type GitHubStep = {
```

Comment:
Is there any reason you guys are waiting to use the octokit packages? It bothers me that we're adding more technical debt that we know we're going to switch soon anyways

Reply:
I'm in the process of converting it now.  There's been some hiccups so it's taking a little longer than I thought.  Should be able to get it done today.

Reply:
I would think new code (aka this PR) could use it without needing to convert old stuff